.ci/bump-release-version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+CS_RELEASE_VERSION="$(getCheckstylePomVersion)"
+echo CS_RELEASE_VERSION="$CS_RELEASE_VERSION"
+
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name" | sed 's/checkstyle-//' | sed 's/"//' | sed 's/"//')
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
+
+SET_VERSION="$LATEST_RELEASE_TAG-SNAPSHOT"
+echo SET_VERSION="$SET_VERSION"
```

Comment:
remove all of this.
you donot need old version to set new version
look at https://github.com/checkstyle/checkstyle/wiki/How-to-make-a-release#before-starting
I never did this during numerous releases.

Reply:
Changes Done